.rustfmt.toml

```diff
@@ -1,3 +1,4 @@
+edition = "2018"
```

Comment:
did this make any difference?

Reply:
When I ran `rustfmt` to resolve the formatting check error in your tests, it complained about the `async fn's`. After specifying this edition the error went away